app/dashboards/user_dashboard.rb

```diff
@@ -94,8 +78,9 @@ class UserDashboard < Administrate::BaseDashboard
 
   # Overwrite this method to customize how users are displayed
   # across all pages of the admin dashboard.
-  #
-  # def display_resource(user)
-  #   "User ##{user.id}"
-  # end
+
+  def display_resource(user)
+    return user.email unless user.first_name.present? && user.last_name.present?
```

Comment:
switch this to `user.email` only.

Reply:
actually `"#{user.first_name} #{user.last_name} <#{user.email}>"`

Reply:
@achapm I ended up just sticking with email - the combined version was just a little messy. We can definitely revisit this down the line.